code/game/machinery/vending.dm

```diff
@@ -3014,6 +3016,7 @@ var/global/num_vending_terminals = 1
 		/obj/item/weapon/reagent_containers/food/drinks/bottle/holywater = 1,
 		/obj/item/clothing/head/helmet/knight/templar = 2,
  		/obj/item/clothing/suit/armor/knight/templar = 5,
+		/obj/item/weapon/gun/hookshot/whip/vampkiller = 1,
```

Comment:
are you sure you want to add a weapon to the pietyvend?

Reply:
It does 1 damage, if that's a problem I can make it do 0.